src/app.jsx

```diff
@@ -352,6 +352,7 @@ class Application extends React.Component {
         case 'create':
         case 'died':
         case 'exec_died':
+        case 'health_status':
```

Comment:
But does it do actually anything? When does podman send this event? Every time healthcheck is performed? If so, then this would be rather annoying to receive this event every time and reload container.

Reply:
That is, I'll check when this event is sent more closely tomorrow.

Reply:
a-ha! yes, it does something. It happens every time when there is a new healthcheck result. This [fails consistently](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1353-20230713-173031-9e7430ad-fedora-38-devel/log.html#14) when I *remove* the "Handle 'health_status' container event" patch, and "*add* the "Ignore irrelevant container events" patch (so that there are no other events which would often, but not always, catch up with health updates). So having both patches is overall a better solution IMHO.

Reply:
I had to revert this, see below. It would be the right thing to do on paper, but it's utterly broken.